### ISSUE TITLE
Scrolling behavior with touchpad is too sensitive

### ISSUE THREAD

**Issue by Keith94** [digimezzo/dopamine-windows#815]
When using my laptop trackpad to scroll the albums list, even just a gentle touch moves the scrollbar very quickly. For me, it's way too sensitive and impossible to control.

**Comment by 0xBRM** [comment]
Yeah, he should definitely implement smooth scrolling in the ListBoxes. That is trivial. It is just setting ScrollMode to ItemScrollerScrollModes.Smooth
